tests/config/test_data_config.py

```
@@ -27,16 +43,18 @@ def test_data_config_initialization():
     assert config.chunk_size == 100
 
 
-def test_preprocessing_config_initialization():
+def test_preprocessing_config_initialization(caplog):
     """Test PreprocessingConfig with valid values."""
-    with pytest.raises(ValueError, match="scale"):
+    with pytest.raises(ValueError):
         config = PreprocessingConfig(max_height=256, max_width=256, scale=(0.5, 0.5))
```

Ruff (0.8.2)

Local variable `config` is assigned to but never used

Remove assignment to unused variable `config`

(F841)